app/workers/shipment_worker.rb

```diff
@@ -0,0 +1,15 @@
+class ShipmentWorker
+  include Sidekiq::Worker
+
+  def perform(*args)
+    five_days_later = 432000
+    delivery_date = Time.now + five_days_later
+    
+    User.users_with_subscriptions.each do |user|
+      user.subscription.shipments.create(
```

Comment:
Also I'll need other model methods that filter user's out by their types of subscriptions to create them only when needed